.travis.yml

```diff
@@ -16,9 +16,8 @@ python:
   - "3.4"
 
 install:
-  - pip install --upgrade pip  --use-mirrors
```

Comment:
Sure, why not. What's the main motivation behind upgrading pip itself first?

P.S. I'll also add sudo-false to profit from faster container-based builds.